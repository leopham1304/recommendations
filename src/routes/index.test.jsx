import React from "react";
import Routes from "./";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("Routes", () => {
  const wrapper = shallow(<Routes />);

  it("should render Routes without crashing", () => {
    expect(toJson(Routes)).toMatchSnapshot();
  });
});
