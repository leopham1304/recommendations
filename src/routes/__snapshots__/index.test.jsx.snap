// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render Routes without crashing 1`] = `undefined`;
